dataactcore/scripts/updateCGAC.py

```diff
@@ -20,21 +21,44 @@ def updateCGAC():
     all_users = userDb.session.query(User).all()
     all_submissions = jobDb.session.query(Submission).all()
 
+    users_to_update = []
+
     for user in all_users:
-        user.cgac_code = findCGACCode(all_cgac, user.cgac_code)
+        if user.cgac_code != None:
+            users_to_update.append(user)
```

Comment:
Does it break any part of the broker to still have users with no CGAC?  Will things like the recent submissions route just return an empty list, or will they 500?